tests/testthat/testImportM.COSMIC.R

```diff
@@ -0,0 +1,111 @@
+# testImportM.COSMIC.R
+#
+#
```

Comment:
Add setup and teardown blocks. Turn logging off for all tests, except for one block in which you **specifically** check the log. 

Structure of tests: first test for rejecting faulty parameters, then test for correct input, then test for not creating faulty output if input is corrupted. (eg. incomplete file, spurious characters in assumed numeric columns, NA expected values ...)